packages/backend/src/graphql/custom-resolvers/flow.ts

```diff
@@ -0,0 +1,17 @@
+import { TEMPLATES } from '@/db/storage'
+
+import type { Resolvers } from '../__generated__/types.generated'
+
+type FlowResolver = Resolvers['Flow']
+
+const template: FlowResolver['template'] = async (parent) => {
+  const templateId = parent?.config?.templateConfig.templateId
```

Comment:
This line will break existing pipes
```suggestion
  const templateId = parent?.config?.templateConfig?.templateId
```

Reply:
wah good catch thanks